skfp/model_selection/randomized_scaffold_split.py

```diff
@@ -131,8 +127,10 @@ def randomized_scaffold_train_test_split(
 
     """
     # flake8: noqa: E501
-    train_size, test_size = validate_train_test_sizes(train_size, test_size)
-    scaffolds = _create_scaffolds(data, include_chirality, use_csk)
+    train_size, test_size = validate_and_scale_train_test_sizes(
```

Comment:
I think previous name was ok, no need for `_and_scale` (scaling has particular meaning in ML, after all)